saleor/userprofile/i18n.py

```diff
@@ -37,6 +50,22 @@
     'zip': pgettext_lazy('Address field', 'ZIP code')}
 
 
+class CustomPhonePrefix(PhonePrefixSelect):
```

Comment:
It's a good practice to include the widget type in the class name so in this case, `CustomPhonePrefixSelect` would be a better name. Also, the word "custom" doesn't say much. What makes this widget custom? It would be great if we could reflect the purpose of this class in the name.